Driver List Update (#1)

TaorLegrand opened the pull request:
Updated Driver List with more Legrand UPSs and removed obsolete ones.

TaorLegrand committed:
Driver List Update (#1)

Updated Driver List with more Legrand UPSs

TaorLegrand committed:
Update driver.list.in

zykh reviewed (changes requested):
note that we list devices even if they're no longer on sale (this is not a catalogue, but just a list of past and present devices that can be currently used with NUT), so you should not remove the Meta System entries

TaorLegrand commented:
if so we have to create a new driver called "Legrand_megawhad" for new models and keep the old one "Metasys" for older models, do you agree? if yes i can send the new and old driver sources.

TaorLegrand committed:
Updated Metasys Driver

List of changes:
- removed some RW flags to items that were Read Only 
- added discrimination for Battery SOC Data so we can keep all the models in one driver
